docs/backups.md

```diff
@@ -1,44 +1,80 @@
-# Backups
+# Concepts
```

Comment:
This file became "concept" page explaining all common backup features (schedule, restore etc.)